extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -373,40 +369,42 @@ export class AmpStoryPage extends AMP.BaseElement {
   }
 
   /**
-   * Reads the storyNextUp param and sets it as the auto-advance-after attribute
-   * if this is the last page and there isn't a value set by the publisher.
+   * Reads the storyNextUp param if provided and sets the auto-advance-after
+   * attribute if there isn't a value set by the publisher. The
+   * auto-advance-after attribute will be set to the tagName of the first
+   * video if there is one, or the value of the storyNextUp param otherwise.
    * @private
    */
   maybeSetStoryNextUp_() {
     const autoAdvanceAttr = this.element.getAttribute('auto-advance-after');
     const storyNextUpParam = Services.viewerForDoc(this.element).getParam(
       'storyNextUp'
```

Comment:
Can we change this param's name? I'd like to make sure we know it's used for user testing and will ultimately ship under another name once we figure out what the right duration is.

Reply:
It can be changed, but I want to push back since that will require us to update and rollout changes to the player client, which will further limit the pool of users we can divert into the experiment. Since this is meant to be a temporary experiment, I'm tempted to leave it as is and make sure we wipe it once we finish experimenting.

With this in mind, though, let me know if you feel strongly about changing it. "autoAdvanceTestDuration" or something sounds nice.

Reply:
Sounds good. Can you please add a code comment to specify that this is a "private" parameter that will either change or be removed at our convenience? Adding this in the code and in the PR description would be nice.

Reply:
Done.